packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -599,6 +599,7 @@ export type CallCompositeOptions = {
     galleryOptions?: {
         layout?: VideoGalleryLayout;
     };
+    survey?: boolean;
```

Comment:
Is there a method for a developer to get the survey results? Will it come through as adapter state or do we need a callback here like
```suggestion
    survey?: boolean | {
      onResultsSubmit: (results: SurveyResults) => Promise<void>
    };
```

Reply:
~~Should it have the ability to choose tag survey vs star survey?~~

```suggestion
    survey?: boolean | {
      onResultsSubmit: (results: SurveyResults) => Promise<void>;
      surveyKind: 'tags' | 'star'
    };
```

Ah I see it is done based on the star rating they provide